src/main/java/org/jfrog/bamboo/util/TaskDefinitionHelper.java

```diff
@@ -116,6 +114,20 @@ public static TaskDefinition findGenericDeployDefinition(List<? extends TaskDefi
         }
         return null;
     }
+    /**
+     * @return Publish build info task if found, null if not.
```

Comment:
@return Publish build info task if found, null if not. --> @return True if a Publish Build Info task exists in the plan. Otherwise, false.

Reply:
Done